code/clue.js

```diff
@@ -41,19 +189,73 @@ const randomSelector = array => {
 // With a killer, a weapon and a room.
 // The values will be set later.
 
+let mystery = {
+  killer:'',
+  weapon:'',
+  room:''
+}
+
+
+
+
 // This function will be invoked when you click on the killer card.
+
 const pickKiller = () => {
   // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
+  
+  // Shuffle weapon function
+  shuffleFavouriteWeapon = () => {
+    mystery.killer.favouriteWeapon = randomSelector(weapons); 
+  };
+    shuffleFavouriteWeapon();
+    console.log(mystery.killer.favouriteWeapon.name)
```

Comment:
No need of this console.log right? Maybe you can remove it.

Reply:
👍